[TXNR-166][PATCH] Adapting the metadata API card documentation (#206)
FidelLimited/docs: cards.md

## Metadata

> The Card object can also have associated metadata, i.e., auxiliary data to better describe or help tracking and working with cards. The metadata field requires an `id` mandatory property, which works as a *non-unique index* for retrieval purposes. We recommend that no confidential or sensitive data regarding the cardholder be stored in clear in this property.

Reviewer: I would break this text down into shorter phrases, and make them more simple. For example (and I hope I don't make wrong assumptions, so please take it with a grain of salt from the technical perspective):
"The metadata field is an object. It must contain a property called `id`, which is a non-unique index. You can use the `id` property to retrieve .... (please add what they can retrieve and how). It's recommended not to include unencoded confidential or sensitive data regarding the cardholder.

Author: @krisztina-nagy based on your comments I adapted the text.

> The Card object can also have associated metadata, i.e., auxiliary data to better describe or help tracking and working with cards. The metadata field is an object and it requires an `id` mandatory property, which works as a *non-unique index* for retrieval purposes. It's recommended not to include unencoded confidential or sensitive data regarding the cardholder.

Reviewer 2: ```suggestion
Cards can also have associated metadata, i.e., auxiliary data to better describe and retrieve cards. The metadata field is an object that requires a mandatory `id` property, which works as a *non-unique index* for retrieval purposes. It's recommended to not store unencoded sensitive data in this field.
```

> When creating a card, you can set the `id` property of the metadata to a custom identifier, for example `my-brand-name-cards` (API Reference [Create Card](https://reference.fidel.uk/reference/create-card) endpoint). You can also update a card's metadata at any time. For more information, refer to the API Reference for the [Update Card Metadata](https://reference.fidel.uk/reference/update-card-metadata) endpoint. Additionally, you can retrieve a list of cards by using the metadata `id`. You can read more about this in our API Reference for the [List Cards from Metadata ID](https://reference.fidel.uk/v1/reference/list-cards-from-metadata-id) endpoint.

Reviewer: Instead of "(API Reference [Create Card](https://reference.fidel.uk/reference/create-card) endpoint)" I would add the link to the first sentence this way:

"When creating a card with the [Create Card](https://reference.fidel.uk/reference/create-card) endpoint, ...."

or:

"When [creating a card](https://reference.fidel.uk/reference/create-card), ..."

The same applies to the update and retrieve:
"You can also update a card's metadata by using the [Update Card Metadata](https://reference.fidel.uk/reference/update-card-metadata) endpoint. Additionally, you can retrieve the list of cards that have a specific value in the metadata `id` field, by using the [List Cards from Metadata ID](https://reference.fidel.uk/v1/reference/list-cards-from-metadata-id) endpoint."

Author: @krisztina-nagy based on your comments I adapted the text.
